examples/apps/presence-tracker/src/FocusTracker.ts

```diff
@@ -48,7 +48,7 @@ export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
 		/**
 		 * A states workspace that the FocusTracker will use to share focus states with other session clients.
 		 */
-		// eslint-disable-next-line @typescript-eslint/ban-types -- empty object is the correct typing
+		// eslint-disable-next-line @typescript-eslint/no-empty-object-type, @typescript-eslint/ban-types -- empty object is the correct typing
```

Comment:
I know this comment says `empty object is the correct typing` but the linter is objecting because `{}` is not an empty object. Even numbers and strings are allowed by {}.  See docs for the lint rule this is suppressing: https://typescript-eslint.io/rules/no-empty-object-type/

I know this isn't new logic, so not an issue with this PR, but seeing a lint rule suppressed with the reason given being the exact misunderstanding that the lint rule exists to prevent disappoints me. We really shouldn't suppress lint rules without reading why the rule exists....